src/App.tsx

```diff
@@ -1,61 +1,150 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todoList, setTodo] = useState(todosFromServer);
+  const [basicData, setData] = useState({
```

Comment:
what is `basicData`?  use better variable names like `todo`

Reply:
I've changed name of variable